aucoalesce/coalesce.go

```diff
@@ -454,6 +472,18 @@ func applyNormalization(event *Event) {
 		return
 	}
 
+	event.ECS.Event.Category = norm.ECS.Category.Values
+	event.ECS.Event.Type = norm.ECS.Type.Values
+
+	hasAdditionalNormalization := syscallNorm != nil && syscallNorm != norm
+	if hasAdditionalNormalization {
+		event.ECS.Event.Category = append(event.ECS.Event.Category, syscallNorm.ECS.Category.Values...)
+		event.ECS.Event.Type = append(event.ECS.Event.Type, syscallNorm.ECS.Type.Values...)
+		if event.Result == "fail" {
+			event.ECS.Event.Outcome = "failure"
```

Comment:
we're only explicit if something was a `failure` otherwise, `unknown`/`success` are implied based off of the context of the category/type